### DISCUSSION TITLE
Use a template for email sent when adding speaker to a proposal

### DISCUSSION THREAD

**Discussion by amottier** [pretalx/pretalx#1121] [Ideas]
Currently the content of the email sent when adding a new user to a proposal is defined in the code (https://github.com/pretalx/pretalx/blob/master/src/pretalx/orga/views/submission.py#L72). It would be nice if this email rely on a template defined in `https://pretalx.com/orga/event/<event name>/mails/templates/`

**Comment by rixx** [comment]
That's a great idea, thank you for your feedback. I can't prioritise it right now, but if anybody else wants to tackle it, I'll be glad to help.
